public/themes/custom/hdbt_subtheme/src/js/matomo.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line func-names
+(function ($, Drupal) {
+  function loadMatomoAnalytics() {
+    if (typeof Drupal.eu_cookie_compliance === 'undefined') {
+      return;
+    }
+
+    // Load Matomo only if statistics cookies are allowed.
+    if (Drupal.eu_cookie_compliance.hasAgreed('statistics')) {
```

Comment:
Mulla tämä kohta ei ikinä palauttanut "TRUE", vaikka hyväksyin kaikki evästeet. Toimiiko toi eu_cookie_compliance toivotulla tavalla?

Reply:
Juuri kokeilin ja minulla tulee true ihan oikein tuosta. Testaathan matomo.min.js:ää, etkä matomo.js:ää kun debuggailet livenä

Reply:
Tän pitäisi toimia jos on hyväksynyt kaikki evästeet.

Jos käyt samalla selaimella nyt stagella icognitona, hyväksyt kaikki evästeet niin saatko noita pk-alkuisia matomo-evästeitä sieltä? (eli toimiiko sulla toi sama scripti jos se ladataan pääteemasta)

Reply:
Joo, kyllä nää tänne ilmestyy:

![Screenshot 2023-11-07 at 16 03 13](https://github.com/City-of-Helsinki/hel-fi-drupal-grants/assets/117807015/3b34692c-26a4-43f6-9ea0-59cd5e6601f5)

Reply:
Toimii.